students/Harry_Maher/session06/test_mailroom4.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python3
+"""
+pytest for mailroom 4.
+
+"""
+# Since I wrote mailroom4, I'm ok w/ importing *
+from mailroom4 import * 
```

Comment:
Yeah, nice!

Reply:
Or you could just say: 

import mailroom as mr (or however you want to alias it)